### DISCUSSION TITLE
How to create .github/workflows in my Github Repository?

### DISCUSSION THREAD

**Discussion by ZedZipDev** [community/community#57945] [Actions] [answered]
Workflows

Question

### Body

I create this folder in browser , it works but finally I see 2 folders created

.github
    workflows
but I need .github/workflows as described it the documentation. I cannot rename it now. 
How to create .github/workflows folder?

**Comment by DSDmark** [comment, marked as answer]
Hi @ZedZipDev,

If you have created two separate folders named **.github** and **workflows** instead of a single folder named `.github/workflows` like this by simply trying that name:

![image](https://github.com/community/community/assets/85744816/b1357682-c97c-4b21-91c6-40a2f5f9e7ea)

If you're still facing that issue, then there is a simple way to create it. For your understanding:

Just click on **Action** tab on your repo in which you want to create.

![image](https://github.com/community/community/assets/85744816/c07005fe-5f5e-48d9-8632-16f594678608)

Now then, it shows something's like that:

![image](https://github.com/community/community/assets/85744816/071bab0e-60e6-437d-b64c-8cdb70d11e3c)

Click to **set up a workflow yourself** and commit it. It will create that folder. Like this:

![image](https://github.com/community/community/assets/85744816/3551321d-0c04-4e71-b896-325ef3f5da04)

> The `.github/workflows` directory is a specific directory structure used in GitHub repositories to store workflow configuration files. GitHub Actions is a powerful feature of GitHub that allows you to automate various tasks and workflows within your repository. Workflow configuration files define the steps, events, and conditions for running automated processes, such as building, testing, and deploying your code. When you create a **.github/workflows** directory in your repository and place workflow configuration files inside it, GitHub recognizes this structure and treats those files as workflow definitions. These workflow files are written in YAML format and define the actions to be performed when certain events occur, such as pushing code to the repository or creating a pull request.

**Reply by ZedZipDev** [reply]
Thanks @DSDmark ,
I have created one folder but really it is 2 but anyway: it works.
